README.md

```diff
@@ -72,7 +72,7 @@ contribute to a package, see the README of the corresponding package.
 - [Johannes Bugiel](https://github.com/wichniowski)
 - [Mike Mitchell](https://github.com/m10l)
 
-You can also reach out using the Contentful community Slack. We've setup a channel [#forma36](https://contentful-community.slack.com/messages/CFXGTMB98) in which we announce latest changes and updates.
+You can also reach out using the [Contentful community Slack](https://www.contentful.com/slack/). We've setup a channel [#forma36](https://contentful-community.slack.com/messages/CFXGTMB98) in which we announce latest changes and updates.
```

Comment:
Wow! Thanks for that

I would just make a small suggestion, what do you think about moving this up? Like:

```
## Reach out to us

You can reach out to us using the [Contentful community Slack](https://www.contentful.com/slack/)...

### Maintainers
....
```

Reply:
Yeah I think that makes sense. I guess the idea is that you would want people to reach out via the Slack channel before they went straight to a maintainer correct?

I'll push up another commit after work.

Reply:
Yes... Probably we will add more communication channels here, but that's a good start I would say